<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fyler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="topGrid">
     <item row="0" column="0">
      <widget class="QLabel" name="sourceTitle">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Original Path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="destList"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="destTitle">
       <property name="text">
        <string>New Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="sourceList"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="sourceHorizontal">
       <item>
        <spacer name="sourceLeftSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sourceUpButton">
         <property name="text">
          <string>ü°Ö</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sourceDownButton">
         <property name="text">
          <string>ü°á</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sourceXButton">
         <property name="text">
          <string>‚ùå</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sourceRightSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="destHorizontal">
       <item>
        <spacer name="destLeftSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="destUpButton">
         <property name="text">
          <string>ü°Ö</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="destDownButton">
         <property name="text">
          <string>ü°á</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="destXButton">
         <property name="text">
          <string>‚ùå</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="destRightSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="bottomGrid">
     <item row="3" column="0">
      <widget class="QPushButton" name="filesButton">
       <property name="text">
        <string>Add Files</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="folderButton">
       <property name="text">
        <string>Add Folder</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="matchButton">
       <property name="text">
        <string>Match</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="actionButton">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="settingsButton">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="advancedButton">
       <property name="text">
        <string>Advanced Tools</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
